lib/sprite.dart

```diff
@@ -10,11 +11,11 @@ class Sprite {
 
   Sprite(
     this.image, {
-    Vector2 position,
+    Vector2 srcPosition,
     Vector2 size,
```

Comment:
since the other one is srcPosition, how about srcSize? to distinguish it from originalSize which is the whole image size? (just a thought, not sure if it's needed)

Reply:
Sounds reasonable, fixed